package/obs/rmt-server.changes

```diff
@@ -17,6 +17,7 @@ Tue Jan 21 10:49:54 UTC 2025 - Jesús Bermúdez Velázquez <jesus.bv@suse.com>
     * Fix Migration from Micro 5.5 to Micro 6.X (bsc#1236816)
     * Add registry optional (bsc#1237373)
   * Update Puma webserver
+  * Update rack (bsc#1242898)
```

Comment:
wouldn't this need to be on the `2.23` section of the changelog ?

Reply:
The Rack update happened in 2.22, but yes we can move it up to 2.23 to avoid issue with automated maintenance changelog checks.